.gitignore

```diff
@@ -1,3 +1,7 @@
+node_modules
+styles.scss
+styles.css.map
+
 # Logs
 logs
 *.log
```

Comment:
- [Optional] kindly, when you have unnecessary files don't add them all in  `.gitignore` delete them, because it makes your code clean. 💯 👍